app/views/admins/pages/_orders.html.erb

```diff
@@ -21,7 +21,11 @@
         <td><%= order.reward.description %></td>
         <td><%= time_ago_in_words(order.created_at) %> ago</td>
         <td><%= order.purchase_price %></td>
-        <td><%= order.employee.email %></td>
+        <% if order.employee.full_name.present? %>
```

Comment:
I added it to the employee model to full_name method:

```
  def full_name
    return [first_name, last_name].join(' ') if (first_name && last_name).present?

    email
  end
```

but since I don't need it (all employees have first and last names) I decided for earlier versions (without email option in case of no first and last name).